config/overlays/tls/patches/manager_deployment_cert.yaml

```diff
@@ -0,0 +1,9 @@
+- op: add
+  path: /spec/template/spec/volumes/-
+  value: {"name":"ca-certificate", "secret":{"secretName":"catalogd-catalogserver-cert", "optional": false, "items": [{"key": "tls.crt", "path": "tls.crt"}]}}
```

Comment:
Do you know if the file contents in the volume mount change if the secret contents change?

Reply:
I just verified that the contents of the mounted file /var/certs/tls.crt on the filesystem do change after manually rotating the certificate.

Reply:
That's good. That's step 1 of being able to rotate the CA cert. But it seems moot for now since there's no easy in-process way to rotate them with an existing server/listener.

This is a good detail to capture in the "let's handle CA cert rotation" issue though.